development.ini

```diff
@@ -15,8 +15,15 @@ pyramid.includes =
     pyramid_debugtoolbar
     pyramid_celery
 
-git.path = %(here)s/repo/
-git.content_repo_url = https://github.com/universalcore/unicore-cms-content-ffl-tanzania
+# git.path can be a local path or a URL that points to a unicore.distribute repo endpoint.
+# The index on Elasticsearch corresponds to the basename (minus .git or .json suffix).
+# For that reason, the basename must be alphanumeric and lowercase. It may also not
+# start with _.
+git.path = %(here)s/repo
```

Comment:
Can we get rid of this and use `git.content_repo_url` in the same way that springboard does?